packages/common/solana/types.ts

```diff
@@ -1,5 +1,11 @@
-import { AccountInfo, BlockhashWithExpiryBlockHeight, Commitment, Context, PublicKey } from "@solana/web3.js";
-import PQueue from "p-queue";
+import {
```

Comment:
```suggestion
import type {
```
Why not just use 1 type keyword?

Reply:
there is no particular reason. 
Perhaps, that happened automatically because auto-imports don't treat this as a meaningful difference. 
I couldn't make an IDE (and I tried only mine, there might be other differences in other apps too) import separately 
like

```
import { NON_TYPES } from "package";
import type { ALL_TYPES } from "package";
```

it just ends up as using different syntax depends on from there the auto import called. 

I'm going to update these manually, but other places may have both syntaxes used.